CorsixTH/Lua/dialogs/resizables/file_browsers/save_game.lua

```diff
@@ -75,6 +76,7 @@ function UISaveGame:confirmName()
   local filename = self.new_savegame_textbox.text
   local app = self.ui.app
   if filename == "" then
+    self.ui:addWindow(UIInformation(self.ui, { _S.save_game_window.missing_filename }))
```

Comment:
I'd write the table without spaces inside the curly braces (also at line 84). Not sure what the convention is.

Also not too important, I am fine if you prefer this.

Reply:
Convention appears to be to use spaces (at least according to `app.lua`) inside, I probably just followed the example.
I'll commit the other line break suggestion though you put in elsewhere.